docs/blog/2019-09-11-gazette-september/index.mdx

```diff
@@ -0,0 +1,106 @@
+
+Marcy also presented a full-length workshop on [Accessibility in JavaScript Applications for Frontend Masters](https://frontendmasters.com/workshops/javascript-accessibility/), including some exclusive tidbits about building accessible Gatsby sites. In case you haven't read her post or seen her workshop already, please _do_ to learn how to make your web apps more accessible.
+
+### GraphQL
+
+- Abstract data interfaces
+- Materialisation
+- Schema snapshots feature added
+
+### Offline
+
+[David](https://twitter.com/davidbailey00) (who recently joined the Gatsby core team) has been working on [gatsby-plugin-offline](https://www.gatsbyjs.org/packages/gatsby-plugin-offline/) for a while and recently released v3.
+
+v3 includes updated Workbox,
+
+### GraphiQL Explorer now available in WPGraphQL
+
+### Optimized JavaScript bundle configuration
+
+### Renovate
+
+Enabled automatic dependency updates for the Gatsby repo
+
+## Community Highlights
+
+It's been a pretty busy month on the monorepo. We've merged in a _lot_ of your contributions and they've all been amazing. Let's go over some of the highlights!
+
+### GraphiQL Code Explorer
+
+Back in June, [Michal](https://twitter.com/mipiechowiak) from the Gatsby core team worked on adding [GraphiQL Explorer](https://github.com/OneGraph/graphiql-explorer) to GraphiQL in Gatsby. One of the next steps he mentioned was the addition of code snippet generation using graphiql-code-exporter.
+
+Gatsby now includes Code Explorer in GraphiQL out of the box starting from v2.15.3, thanks to [Dan Kirkham](https://twitter.com/herecydev).
+
+![GraphiQL Code Explorer](./2019-09-1308.05.19.gif)
+
+When you write a query in GraphiQL, this will automatically generate code for you! You can check out Dan's work on [the PR on GitHub](https://github.com/gatsbyjs/gatsby/pull/17120).
+
+### Notable PRs
+
+It's _really_ hard to pick out favourites from all the incredible contributions you folks make so apologies in advance if I missed anything (let [us](https://twitter.com/gatsbyjs) know on Twitter). Here are my favourites from the past month:
+
+- [Jeremy Albright](https://github.com/Js-Brecht) did some really solid work and fixed an issue where Gatsby CLI would leave your terminal in an inconsistent state in [#16714](https://github.com/gatsbyjs/gatsby/pull/16714)
+- [Nicolas Goutay](https://github.com/phacks) fixed a memory leak in IntersectionObservers (with help from [atomiks](https://github.com/atomiks)) in [#17056](https://github.com/gatsbyjs/gatsby/pull/17056)
+- [Benedikt Franke](https://github.com/spawnia) brought order to some chaos by making root plural fields non nullable in [#15321](https://github.com/gatsbyjs/gatsby/pull/15321)
+- [Mateusz Burzyński](https://github.com/Andarist) cleaned up several significant parts of Gatsby by refactoring some code paths over to XState in [#17192](https://github.com/gatsbyjs/gatsby/pull/17192)
```

Comment:
just to clarify - it has been using XState already, I've just upgraded it to XState@4, refactored a little bit to use newer APIs, so the visualization is way better than what could be achieved with older implementation

Reply:
Yup, thank you once again for all of that!